index.html

```diff
@@ -1,232 +1,395 @@
+            </li>
+            <li>
+              <a
+                href="https://www.linkedin.com/in/sharon-odhiambo-4333a0163/"
+                target="_blank"
+                rel="noopener"
+                ><img src="./images/github.svg" alt="LinkedIn" />
+              </a>
+            </li>
+            <li>
+              <a
+                href="https://www.linkedin.com/in/sharon-odhiambo-4333a0163/"
+                target="_blank"
+                rel="noopener"
+                ><img src="./images/medium.svg" alt="LinkedIn" />
+              </a>
+            </li>
+            <li>
+              <a
+                href="https://github.com/sharon-odhiambo"
+                target="_blank"
+                rel="noopener"
+                class="social-media-link"
+                ><img src="./images/angelist.svg" alt="Github" />
+              </a>
+            </li>
+          </ul>
+          <div>
+            <button type="button" class="button">
+              <span>Get my Resume</span>
+            </button>
+          </div>
+        </div>
+        <div class="skill-section-items">
+          <ul>
+            <li>
+              <div class="languages-dropdown-bar">
+                <p>
+                  Langauges<span
+                    ><img src="./images/dropdown..png" alt="languages"
+                  /></span>
+                </p>
+                <div class="skill-items">
+                  <ul>
+                    <li>
+                      <span
+                        ><img
+                          src="./images/JS.png"
+                          alt="js-icon"
+                        />Javascript</span
+                      >
+                    </li>
+                    <li>
+                      <span
+                        ><img
+                          src="./images/HTML.png"
+                          alt="html-icon"
+                        />HTML</span
+                      >
+                    </li>
+                    <li>
+                      <span
+                        ><img src="./images/CSS.png" alt="css-icon" />CSS</span
+                      >
+                    </li>
+                  </ul>
+                </div>
+              </div>
+            </li>
+            <li>
+              <div class="skill-projects-item-one">
+                <p>
+                  Frameworks<span
+                    ><img src="./images/arrow.png" alt="frames-icon"
+                  /></span>
+                </p>
+              </div>
+            </li>
+            <li>
+              <div class="skill-projects-item-two">
+                <p>
+                  Skills<span
+                    ><img src="./images/arrow.png" alt="skills-icon"
+                  /></span>
+                </p>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </section>
+    <section class="form">
+      <form action="/my-form-page" method="post">
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly ensure you've used the right [formspree](https://formspree.io/html) to enable the submission of a form on your portfolio website, right now, your form is not submitting as expected.